lib/utils/createElement.js

```diff
@@ -9,8 +10,8 @@ export function toggleButton(
 	onClick?: (enabling: boolean) => void = () => {},
 	fieldID: string,
 	enabled?: boolean = false,
-	onText?: string | HTMLElement = 'on',
-	offText?: string | HTMLElement = 'off',
+	onText?: string | HTMLElement = i18n('toggleOn'),
```

Comment:
This is all run synchronously so I think I prefer it this way--if callers who want to provide different text are forced to call `i18n` there will be less implicit behaviour, and unlike module descriptions, etc. there won't be any extra `i18n` calls.